crates/bevy_ui/src/measurement.rs

```diff
@@ -23,6 +23,9 @@ pub struct MeasureArgs<'a> {
     pub available_height: AvailableSpace,
     #[cfg(feature = "bevy_text")]
     pub font_system: &'a mut bevy_text::cosmic_text::FontSystem,
+    // When `bevy_text` is disabled, use `PhantomData` in order to keep lifetime in type signature.
+    #[cfg(not(feature = "bevy_text"))]
+    pub font_system: std::marker::PhantomData<&'a ()>,
```

Comment:
Question: possibly a dumb question haha but does using `&'a ()` instead of `&'a mut ()` have any meaningful effect?

Reply:
[Consulting the nomicon](https://doc.rust-lang.org/nomicon/phantom-data.html#table-of-phantomdata-patterns), it looks like it changes nothing.

Reply:
Ooo, it looks like it actually changes the variance. Let me make that `mut` real quick...